.ci/pipeline/ci.yml

```diff
@@ -57,6 +57,57 @@ jobs:
       pip install isort black[jupyter]==24.1.1
       isort --profile black --check . && black --check .
     displayName: 'Linting'
+- job: LinuxCondaRecipe
+  dependsOn: Lint
+  timeoutInMinutes: 120
+  strategy:
+    matrix:
+      Python3.11:
+        PYTHON_VERSION: '3.11'
+  pool:
+    vmImage: 'ubuntu-22.04'
+  steps:
+  - script: |
+      bash .ci/scripts/install_dpcpp.sh
+    displayName: "dpcpp installation"
+  - script: |
+      conda config --add channels conda-forge
+      conda config --set channel_priority strict
+      conda update -y -q --all
+    displayName: "Conda update"
+  - script: |
+      conda create -y -q -n build-env python=$(PYTHON_VERSION) conda-build conda-verify
+    displayName: "Conda install"
+  - script: |
+      export DPCPPROOT=/opt/intel/oneapi/compiler/latest
+      . /usr/share/miniconda/etc/profile.d/conda.sh
+      conda activate build-env
+      conda build .
```

Comment:
It make sense build with no tests (or just run package import test) and run functional tests separately in other step. What do you think?

Reply:
Doesn't make sense: more steps and script lines in CI, reading logs doesn't become easier anyway.